dir.targets

```diff
@@ -154,5 +154,12 @@
 
   </Target>
 
+  <Target Name="SetPublicReleaseVersion" AfterTargets="GetBuildVersion">
+    <PropertyGroup Condition=" '$(PublicRelease)' == 'true' ">
+      <!-- Removes the prerelease tag from version when this is a public version but still has the git commit-->
+      <AssemblyInformationalVersion>$(BuildVersionSimple)$(SemVerBuildSuffix)</AssemblyInformationalVersion>
```

Comment:
This is stripping out the pre-release tag `-preview` which GitVersioning does not do.

Without this:
```C#
[assembly: AssemblyInformationalVersion("15.1.543-preview+gdd5e8bc3f8")]
```

As far as I can tell, `PublicRelease` is about removing the git commit from stuff like the NuGet package.  The pre-release tag is never removed from what I could tell.